site-container.yml.sample

```diff
@@ -559,7 +559,7 @@
           name: ceph-grafana
       when: dashboard_enabled
 
-- hosts: '{{ (groups["mgrs"] | default(groups["mons"]))[0] }}'
+- hosts: '{{ (groups["grafana-server"] | default(groups["mgrs"]) | default(groups["mons"]))[0] | default(omit) }}'
```

Comment:
If you have an inventory in which the group is defined but no hosts are present inside it, according to this line the group is matched and the playbook is skipped (with the `skipping: no hosts matched` message).

For this reason, if you want to maintain this format, the `hosts` line should contain a size check like:
```
hosts: '{{ ((groups["grafana-server"] if groups["grafana-server"]|length>0) | default(groups["mgrs"] if groups["mgrs"]|length>0) | default(groups["mons"]))[0] | default(omit) }}'
```

and the execution should be fine.